convene-web/app/views/workspaces/show.html.erb

```diff
@@ -9,7 +9,7 @@
 
 <div class="p-6">
   <ul class="mt-3 grid grid-cols-1 gap-5 sm:gap-6 sm:grid-cols-2 lg:grid-cols-4">
-    <% @workspace.rooms.each do |room| %>
+    <% @workspace.rooms.accessable_by.each do |room| %>
```

Comment:
I think we still want to pass in `current_user` even if we're not using it for anything and it's a `nil` value. That way when we do start implementing users we don't have to find and update usages of accessable by.

Thoughts?

Reply:
Right, but I believe we don't have `current_user` yet.

Reply:
We could add a `helper_method def current_user; nil; end` to `ApplicationController` as a seam if we want.